helios/pipeViewer/pipe_view/core/src/core.pyx

```diff
@@ -203,6 +200,9 @@ cdef wxFont* getFont(font):
 cdef wxBrush* getBrush(brush):
     return getBrush_wrapped(<PyObject*>brush)
 
+# Color expression namespace
+EXPR_NAMESPACE = {'re':re, 'colorsys':colorsys, 'math':math, 'extract':extract_value}
+
```

Comment:
Just needed to move this after `extract_value` was defined - I have no idea why this ever worked to begin with.

Reply:
That was my temporal fix as well!